tracing-subscriber/src/registry/sharded.rs

```diff
@@ -89,6 +89,17 @@ fn id_to_idx(id: &Id) -> usize {
     id.into_u64() as usize - 1
 }
 
+// CloseGuard is used to track how many Registry-backed Layers have
```

Comment:
Nit/TIOLI: RustDoc comments usually don't start with the type names...

```suggestion
// A guard that tracks how many Registry-backed Layers have
```